patch/savefile.txt

```diff
@@ -0,0 +1,202 @@
+PatchSavefile:
+	call CheckNotEmpty
+	call CheckSavefileIsValid
+	call PrintSavefileMessage
+	; set #address, VersionInfo
+	; getbyteinc #result, #address
+	; getbyte #temp, #address
+	; mulacum #temp, #result, 0x100
+	set #temp, CURRENT_BUILD
+	ifgt #build, 9999, .invalid_build
+	ifgt #build, #temp, .future_build
+	ifeq #build, #temp, .latest_build
+	push #temp
+	bufstring .savefile_build_string_1
+	set #1, #build
+	set #2, 4
+	call PrintWithLeadingZeros
+	bufstring .savefile_build_string_2
+	printbuf
+	menu #result, YesNoMenu
+	jumpnz #result, .refused
+	; call PokecenterCheck
+	set #address, SavePatches
+.loop
+	gethalfwordinc #curbuild, #address
+	bufchar 0x76
+	bufnumber #curbuild
+	printbuf
+	jumpz #curbuild, .done
+	getwordinc #patch, #address
+	ifle #curbuild, #build, .loop
+	push #address
+	call #patch
+	pop #address
+	jump .loop
+.done
+	pop #build
+	set #curbuild, #build
```

Comment:
As far as I can tell, this is a dead store — nothing's using `#curbuild` anymore

Reply:
`FixSavefileChecksum` calls `GetGameDataPointer` which takes in `#curbuild` as a parameter. Whether that's necessary or desirable is open for debate, but it fits what the code is trying to do there.